docs/home/howto/PREREQUISITES.md

```diff
@@ -60,45 +60,6 @@ docker run -it -v LOCAL_DIR:/shared --rm epiphanyplatform/epicli:TAG
 
 Where `LOCAL_DIR` should be replaced with the local path to the directory for Epicli input (SSH keys, data yamls) and output (logs, build states).
 
-## Run Epicli directly from OS
```

Comment:
Why that section got deleted?

Reply:
Currently not all needed dependencies are Python based anymore. For running test for example we require Rust + additional dependencies that need to be installed. So for now the container approach is the only one we want to advertise.

Reply:
ok